src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -114,14 +113,6 @@ public function setAclProvider(AclProviderInterface $aclProvider)
         $this->aclProvider = $aclProvider;
     }
 
-    /**
-     * @param IndexablePagePartsService $indexablePagePartsService
-     */
-    public function setIndexablePagePartsService(IndexablePagePartsService $indexablePagePartsService)
```

Comment:
The property is protected, we should keep this set and remove it in 6.0. See comment above